### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ShualLiu**
I think there is something wrong with the word 'is', so delete it.

### CODE REVIEW

**Comment by MakMukhi** [comment]
Thanks for the change.

**Comment by ShualLiu** [comment]
I signed it!

File: examples/README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ OPTIONAL - Rebuilding the generated code
 
 1 First [install protoc](https://github.com/google/protobuf/blob/master/README.md)
   - For now, this needs to be installed from source
-  - This is will change once proto3 is officially released
+  - This will change once proto3 is officially released
```
Comment (menghanl) [inline examples/README.md]: This is actually not true anymore...

Can you remove these two lines and update the link for `install protoc`?
```markdown
First [install protoc](https://github.com/google/protobuf/blob/master/README.md#protocol-compiler-installation)
```

Comment (ShualLiu) [reply examples/README.md]: It's my pleasure.

**Pull request merged**